lib/dgram.js

```diff
@@ -134,7 +135,7 @@ function replaceHandle(self, newHandle) {
   self._handle = newHandle;
 }
 
-Socket.prototype.bind = function(port_ /*, address, callback*/) {
+Socket.prototype.bind = promisify(function(port_ /*, address, callback*/) {
```

Comment:
@jasnell Yep, I tried to get everything but missed some bits — I am tracking that over [here](https://github.com/chrisdickinson/node-1/issues/1).